styles.css

```diff
@@ -0,0 +1,875 @@
+/* --------------- 3.1 Reusables ----------------------- */
+
+.outer-container {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    /*border: 1px solid black;*/
+    flex-grow: 0;
+    flex-shrink: 1;
+}
+
+.inner-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    /*border: 1px dotted blue;*/
+    text-align: center;
+
+    flex-basis: 425px;
+    flex-grow: 0;
+    flex-shrink: 1;
+    margin: 0 20px;
+
+}
+
+
+/* --------------- 3.2 Other --------------------------- */
+
+/* ------------------------------- 4. Areas ---------------------------------------- */
+
+/* --------------- 4.1 Header ------------------------- */
+
+.outer-container  {
+    background-color: white;
+}
+
+.inner-container    {
+    /*background-color: var(--main-color-blue);*/
+    color: var(--main-bg-color);
+}
+
+.header-content  {
+    background-color: hsl(242, 75%, 59%);
+    background-image: url(assets/noise-texture.png);
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin: 0 20px;
+
+
+    outline: white;
+    /*background: rgb(9,80,121);*/
+    /*background: linear-gradient(90deg, rgba(9,80,121,1) 0%, rgba(2,0,36,1) 0%, rgba(0,212,255,1) 100%);*/
+}
+
+.nav-container  {
+    width: 100%;
+    margin-bottom: 50px;
+}
+
+.nav-container  ul {
+    list-style-type: none;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 16px;
+}
+
+.nav-container ul a {
+    text-decoration: none;
+    color: var(--main-bg-color);
+    font-weight: bold;
+    font-size: 20px;
+}
+
+.header-content h1   {
+    padding: 20px;
+}
+
+.header-text    {
+    margin-bottom: 16px;
+}
+
+.header-text p  {
+    font-size: 20px;
+    font-weight: lighter;
+    color: white;
+}
+
+#intro-button  {
+    background-color: #ef4183;
+    color: white;
+    font-family: 'Roboto', sans-serif;
+    font-weight: normal;
+    font-size: 16px;
+    border-radius: 30px;
+    margin-top: 16px;
+    margin-bottom: 100px;
+    padding: 16px 30px;
+    border: none;
+    box-shadow: 0 8px 15px rgba(0, 0, 0, 0.2);
+    cursor: pointer;
+}
+
+#triangle-left {
+    width: 0;
+    height: 0;
+    border-top: 50px solid transparent;
+    border-right: 425px solid transparent;
+    border-bottom: 50px solid white;
+    outline: white;
+}
+
+#squiggle-line-item {
+    margin-bottom: -20%;
+}
+
+
+
+/* --------------- 4.2 Introduction ------------------- */
+
+.intro-container    {
+    background-color: white;
+    text-align: center;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    margin-top: 150px;
+    opacity: 0.9;
+}
+
+.intro-container:after  {
+    margin-top: -40px;
+    content: "hello";
+    position: absolute;
+    color: #F7F6FF;
+    font-size: 100px;
+    font-weight: bolder;
+    font-family: 'Roboto', sans-serif;
+    z-index: -1;
```

Comment:
Als je deze hebt toegepast om ervoor te zorgen dat de tekst `"hello"` niet in beeld zou komen, dan zou je ook de content anders kunnen specificeren en is deze z-index wel niet nodig
```
content: "";
```